Documentation/contributing/docs/docsframework.rst

```diff
@@ -0,0 +1,382 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
```

Comment:
This one is the default header used in most source files for the docs. I acknowledge your suggestion, but I'll keep the phrasing for consistency with other files. We may want to fix them all at once, at some point (or it's maybe not worth it, because these headers don't matter much as far as I can tell).

Reply:
That makes sense. My thought process during review:
"Don't we define this header as a global variable? If it's written here, I guess not."

So there's another out of scope idea. 😅

Reply:
No, this header doesn't show in the HTML version, it's only for the rST files, for readers who somehow end up reading the sources. Which means that if we used a variable to define it, nobody would ever see it.